code/common/strings.h

```diff
@@ -20,7 +21,6 @@ namespace common::strings
     std::same_as<type, char> || std::same_as<type, wchar_t>;
   };
 
-#if defined(_WIN32)
```

Comment:
Keep for wstring but sjis needs to work on linux

Reply:
Addressed in a1d45580444fd8a62b1644297758291c587b6198